src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+type Props = {
+  value: number;
+  setValue: React.Dispatch<React.SetStateAction<number>>;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const DropDown: React.FC<Props> = ({ value, setValue, setPage }) => {
+  const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    event.preventDefault();
```

Comment:
The `preventDefault` method is typically used to prevent the default action of an event. However, for the `onChange` event of a select element, there's no default action that needs to be prevented. You can safely remove this line.